src/aiokatcp/connection.py

```diff
@@ -314,19 +314,16 @@ def transform_args(args) -> list:
         msg = args[-1]
         args = list(args[:-1])
         # This relies on zip stopping at the end of the shorter sequence
-        for argument, decoder in zip(msg.arguments, pos_decoders):
-            try:
+        try:
```

Comment:
I wonder what the overhead of the `try .. except` block is.
I was always under the impression it was best practice to keep this as limited as possible, to reduce the likelihood that exceptions other than the one you anticipate come up.

Reply:
Yes, it is generally a good idea. The except statement was showing up in the flamegraph, which is why I did it, but I don't know how accurate that is. On the other hand, I can't see any way the `for` statement could raise a ValueError.